Update and restructure BTP Manager docs (#284)
kyma-project/btp-manager: docs/operations.md

- Secret `sap-btp-manager` with data for SAP BTP Service Operator

The Namespace and PriorityClass resources are created during Kyma installation. The Secret is injected into the cluster
by Kyma Environment Broker (KEB). If you want to provision SAP BTP Service Operator on a cluster without Kyma, you must create
the prerequisites yourself.

### Process

> The provisioning process is part of a module reconciliation. To trigger the reconciliation, create a [BtpOperator CR](../api/v1alpha1/btpoperator_types.go) :

Reviewer: ```suggestion
The provisioning process is part of a module reconciliation. To trigger the reconciliation, create a [BtpOperator CR](../api/v1alpha1/btpoperator_types.go):
```

for performed operations. The provisioning is successful when all module resources exist in the cluster. This is the
condition which allows the reconciler to set the CR in `Ready` state.

> The provisioning process is carried out as presented in the following diagram:

Reviewer: ```suggestion
You can see the provisioning process in the following diagram:
```

Regardless of mode, in the next step, all SAP BTP Service Operator resources marked with the `app.kubernetes.io/managed-by:btp-manager`
label are deleted. The deletion process of module resources is based on resources GVKs (GroupVersionKinds) found in [manifests](../module-resources).
If the process succeeds, the finalizer on BtpOperator CR itself is removed and the resource is deleted.
If an error occurs during the deprovisioning, state of BtpOperator CR is set to `Error`. 

> The deprovisioning process is presented in the following diagram:

Reviewer: ```suggestion
You can see the deprovisioning process in the following diagram:
```
